app/views/eventPermissions.html

```diff
@@ -137,12 +147,13 @@ <h3 translate>Add User</h3>
           type="button"
           class="btn btn-success btn-block"
           ng-click="addPermission(addPermissionsEmail, addPermissionsLevel)"
+          ng-disabled="permissions.$invalid"
```

Comment:
Might be best to use `permissions.$valid` as when the form is in pending state, `permissions.$invalid` is set to FALSE.